robot/Cumulus/tests/browser/gift_entry/reorder_template.robot

```diff
@@ -0,0 +1,109 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             API Check And Enable Gift Entry
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+
+Setup Test Data
+    [Documentation]     Setsup namespace prefix
+    ${ns} =  Get NPSP Namespace Prefix
+    Set suite variable    ${ns}
+
+Get Template Builder Field Names
+  @{BUILDER_FORM_FIELDS} =  Return Template Builder Titles  template_builder_fields
+  ${BUILDER_FIELD_LABELS} =  Create List
+
+  FOR  ${LABEL}  IN  @{BUILDER_FORM_FIELDS}
+      ${NAME} =  Get Text  ${LABEL}
+      Append to List  ${BUILDER_FIELD_LABELS}  ${name}
+  END
+  Set Suite Variable  ${BUILDER_FIELD_LABELS}
+
+Get Template Builder Section Names
```

Comment:
Same here documentation for this one too

Reply:
Done